recipe/meta.yaml

```diff
@@ -38,7 +38,7 @@ test:
     - pyscaffold.templates
   commands:
     - putup --help
-    - pytest -m "not slow and not system" --cov-config setup.cfg
+    - pytest -m "not skip_on_conda_build" --cov-config setup.cfg
```

Comment:
I think here we either leave it with no mark (the `skip_on_conda_build` is automatically calculated based on the env vars), or use the `requires_src` (which is applied to the same places `skip_on_conda_build` is applied, anyway). There is yet the third option of leaving as it is, so we stick with very minimal builds...

Reply:
All options sound good to me. I don't have any particular preference. Feel free to change yourself, or just let me know now what you'd prefer (since I can make the change more easily since I own the branch)